source/documentation/guidance/upgrading-from-cflinuxfs2.md

```diff
@@ -0,0 +1,25 @@
+## Upgrading from `cflinuxfs2`
+
+When you run an app on the PaaS using a buildpack, it runs on top of a version of Ubuntu Linux. We are upgrading the version of Ubuntu used from Ubuntu Trusty to Ubuntu Bionic. In Cloud Foundry, this takes the form of upgrading the Stack your apps use from `cflinuxfs2` to `cflinuxfs3`.
```

Comment:
I would add a link to `https://wiki.ubuntu.com/Releases` e.g. "Refer to the [Ubuntu releases](https://wiki.ubuntu.com/Releases) webpage for more information.

Is `Stack` usually capitalised? We try to minimise capitalisation.

Reply:
> Is Stack usually capitalised? We try to minimise capitalisation.

Stacks are so rarely discussed that I don't know. I've lowercased it.